internal/characters/kinich/burst.go

```diff
@@ -55,40 +61,33 @@ func (c *char) Burst(p map[string]int) (action.Info, error) {
 	}, nil
 }
 
-func (c *char) QueueLaser(step, src int) {
-	if src != c.ajawSrc {
-		return
-	}
-	// duration expired
-	if c.Core.F-c.ajawSrc > ajawDuration {
-		return
+func (c *char) QueueLaser(step, src int) func() {
+	return func() {
+		if c.ajawSrc != src {
+			return
+		}
+		// duration expired
+		if !c.StatusIsActive(burstKey) {
+			return
+		}
+		// condition to track number of hits just in case
+		if step == 7 {
+			c.DeleteStatus(burstKey)
+			return
+		}
+		ai := combat.AttackInfo{
+			ActorIndex:     c.Index,
+			Abil:           "Hail to the Almighty Dragonlord (Dragon Breath DMG)",
+			AttackTag:      attacks.AttackTagElementalBurst,
+			AdditionalTags: []attacks.AdditionalTag{attacks.AdditionalTagNightsoul},
+			ICDTag:         attacks.ICDTagElementalBurst,
+			ICDGroup:       attacks.ICDGroupDefault,
+			StrikeType:     attacks.StrikeTypeDefault,
+			Element:        attributes.Dendro,
+			Durability:     25,
+			Mult:           burst[c.TalentLvlBurst()],
+		}
+		c.Core.QueueAttack(ai, combat.NewSingleTargetHit(c.Core.Combat.PrimaryTarget().Key()), 0, 0)
```

Comment:
One thing I forgot: maybe we should queue attack for all the present enemies. Or maybe give hist laser AOE, despite it's single target.